spec/index.html

```diff
@@ -722,19 +723,25 @@ <h3>Properties of simple entailment and satisfiability</h3>
     
     <p>We define the <dfn>set of propositions</dfn> in an interpretation as follows:</p>
 
-	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}; we observe that a proposition is in the extension of <code>rdfs:Proposition</code>. </p>
+	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}. </p>
+	
+	<p>The denotation of a triple is a proposition, regardless of whether it is used as a triple term or an asserted triple. Given the semantics of <a href="#rdfs_interpretations">RDFS Interpretations</a>, a proposition is in the extension of the class <code>rdfs:Proposition</code>.</p>
 
 	<p>We define the <dfn>set of facts</dfn> in an interpretation as follows:</p>
 
-	<p class="fact"> The set F of facts in an interpretation I is F(I) = {&nbsp;IT(x, y, z)&#65372;&lt;x, z&gt; is in IEXT(y)&nbsp;}. The set of facts is the set of propositions which are true in the interpretation. </p>
+	<p class="fact"> The set F of facts in an interpretation I is F(I) = {&nbsp;IT(x, y, z)&#65372;&lt;x, z&gt; is in IEXT(y)&nbsp;}. </p>
+	
+	<p>A fact in an interpretation is a proposition which is true in the interpretation. In other words, a fact in an interpretation is the proposition corresponding to the denotation of an asserted triple in the interpretation. </p>
```

Comment:
```suggestion
	<p class="fact"> The set F of facts in an interpretation I is F(I)&nbsp;= {&nbsp;IT(x, y, z)&#65372;&lt;x, z&gt;&nbsp;is in IEXT(y)&nbsp;}. </p>
	
	<p>A fact in an interpretation is a proposition which is true in the
	interpretation. In other words, a fact in an interpretation is the proposition
	corresponding to the denotation of an asserted triple in the interpretation.</p>
```

Reply:
Plus, as you pointed out elsewhere, the definition of "set of fact" does not even refer to any graph, so the notion of asserted triple does not even apply. :)